package-lock.json

```diff
@@ -0,0 +1,5131 @@
+{
```

Comment:
Is this really necessary? We use `yarn` for all other packages.

Reply:
I'm not using yarn and this speeds up builds. I wish we could stick with default tools. Yarn should be the one using the package-lock instead of his custom lock file IMO...

https://hackernoon.com/its-ok-to-not-use-yarn-f28dc766ef32

Reply:
But it will be a pain to maintain both a `yarn.lock` and a `package-lock.json` file. Yarn is pre-installed in the Node Docker image and in Travis.

Reply:
Though npm already comes with nodejs ^^. File removed.

Reply:
`yarn.lock` must be updated now 😂

Reply:
```
(master ✗)❯ yarn
zsh: command not found: yarn
```

I don't want it :laughing:. Updated but really this is a pain.